core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -933,14 +933,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
-  public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
-      new Builder(Name.MASTER_HEARTBEAT_INTERVAL_MS)
-          .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms"})
-          .setDefaultValue("10sec")
+  public static final PropertyKey MASTER_MASTER_HEARTBEAT_INTERVAL_MS =
+      new Builder(Name.MASTER_MASTER_HEARTBEAT_INTERVAL_MS)
+          .setAlias(new String[]{"alluxio.master.master.heartbeat.interval.ms"})
+          .setDefaultValue("2min")
           .setDescription("The interval between Alluxio masters' heartbeats.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_WORKER_HEARTBEAT_INTERVAL_MS =
+      new Builder(Name.MASTER_WORKER_HEARTBEAT_INTERVAL_MS)
+          .setAlias(new String[]{"alluxio.master.worker.heartbeat.interval.ms"})
```

Comment:
We don't need `"alluxio.master.worker.heartbeat.interval.ms"` as we don't use it before

Reply:
Changed to 
```
  public static final PropertyKey MASTER_MASTER_HEARTBEAT_INTERVAL =
      new Builder(Name.MASTER_MASTER_HEARTBEAT_INTERVAL)
          .setDefaultValue("2min")
          .setDescription("The interval between Alluxio masters' heartbeats.")
          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
          .setScope(Scope.MASTER)
          .build();
  public static final PropertyKey MASTER_WORKER_HEARTBEAT_INTERVAL =
      new Builder(Name.MASTER_WORKER_HEARTBEAT_INTERVAL)
          .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms",
              "alluxio.master.heartbeat.interval"})
```
Thanks